### PR TITLE
chore(nexus/rebuild): add test assert for rebuild history

### PR DESCRIPTION
**Pull request by dsharma-dc**
Added a simple component test hook to assert that history is not empty.
Detailed test under pytest will be done separately.

Signed-off-by: Diwakar Sharma <diwakar.sharma@datacore.com>

### CODE REVIEW

**Comment by dsharma-dc** [comment]
bors try

File: test/python/v1/rebuild/features/rebuild_history.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Feature: Rebuild history
+    Background:
+        Given the io-engine is running and a volume is created with more than one replicas
```
Comment (tiagolobocastro) [inline test/python/v1/rebuild/features/rebuild_history.feature:4]: why are there 2 different files that seem the same?

Comment (dsharma-dc) [reply test/python/v1/rebuild/features/rebuild_history.feature:4]: Presently I see under tests/ directory two paths where files are duplicated. One directory path is under v1, similar to how grpc code is laid out. I followed that.

Comment (tiagolobocastro) [reply test/python/v1/rebuild/features/rebuild_history.feature:4]: We don't need to update "v0" anymore. Only need to add new features for v1.

Comment (dsharma-dc) [reply test/python/v1/rebuild/features/rebuild_history.feature:4]: Updated accordingly, please take a look. The py file is marked as "to be implemented" which will be completed with GTM-352.

File: test/python/v1/rebuild/features/rebuild_history.feature [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Feature: Rebuild history
+
+    Background:
+        Given the io-engine is running and a volume is created with more than one replicas
+
+    Scenario: Record Full rebuild of a faulted replica
+        Given a volume with multiple replicas, undergoing IO
+        When one of the replica becomes faulted
+        And as a result, a new replica is hosted on a different pool
+        Then a full rebuild of the new replica happens
```
Comment (tiagolobocastro) [inline test/python/v1/rebuild/features/rebuild_history.feature]: Should we be explicit on when a rebuild event is added? eg:

```suggestion
        Then a full rebuild of the new replica starts
        And when the rebuild completes
```

**Review by tiagolobocastro — APPROVED** [review]

File: test/python/v1/rebuild/features/rebuild_history.feature (hunk 2 above)
Comment (dsharma-dc) [reply test/python/v1/rebuild/features/rebuild_history.feature]: Modified

**Review by dsavitskiy — APPROVED** [review]

**Comment by dsharma-dc** [comment]
bors merge

**Review by abhilashshetty04 — APPROVED** [review]

**Pull request merged**